pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,569 @@
+# Create a file (${OUTPUT_FILE}) with all options/values intelligently handled and included
+rm -Rf ${OUTPUT_FILE}
+touch ${OUTPUT_FILE}
+
+while read line; do 
+  OPTION="--$(echo ${line} | awk '{print $1}')"
+  VALUE="$(echo ${line} | awk '{print $2}' | sed 's|^[ \t]*||;s|[ \t]*$||')"
+  if [ "${VALUE}" == "(No" ]; then
+    echoit "Working on option '${OPTION}' which has no default value..."
+  else
+    echoit "Working on option '${OPTION}' with default value '${VALUE}'..."
+  fi
+  # Process options & values
+  if [[ " ${EXCLUDED_LIST[@]} " =~ " ${OPTION} " ]]; then 
+    echoit "  > Option '${OPTION}' is logically excluded from being handled by this script..."
+  elif [ "${OPTION}" == "--enforce-storage-engine" ]; then
+    echoit "  > Adding possible values InnoDB for option '${OPTION}' to the final list..."
+    echo "${OPTION}=InnoDB" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--binlog-error-action" ]; then
+    echoit "  > Adding possible values IGNORE_ERROR, ABORT_SERVER for option '${OPTION}' to the final list..."
+    echo "${OPTION}=IGNORE_ERROR" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ABORT_SERVER" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--enforce-gtid-consistency" ]; then
+    echoit "  > Adding possible values OFF, ON, WARN for option '${OPTION}' to the final list..."
+    echo "${OPTION}=OFF" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ON" >> ${OUTPUT_FILE}
+    echo "${OPTION}=WARN" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--gtid-mode" ]; then
+    echoit "  > Adding possible values OFF, OFF_PERMISSIVE, ON_PERMISSIVE, ON, ON enforce for option '${OPTION}' to the final list..."
+    echo "${OPTION}=OFF" >> ${OUTPUT_FILE}
+    echo "${OPTION}=OFF_PERMISSIVE" >> ${OUTPUT_FILE}
+    echo "${OPTION}=ON" >> ${OUTPUT_FILE}
```

Comment:
I think this line should be removed. IMO, we cannot start the server with --gtid-mode=ON without enabling enforce-gtid-consistency. Please verify and then take appropriate actions.